src/variable.cpp

```diff
@@ -4261,32 +4270,29 @@ Region *Variable::region_function(char *id, int ivar)
      extract_setting(x),label2type(x,y),is_typelabel(x,y)
 ------------------------------------------------------------------------- */
 
-int Variable::special_function(char *word, char *contents, Tree **tree, Tree **treestack,
-                               int &ntreestack, double *argstack, int &nargstack, int ivar)
+int Variable::special_function(const std::string &word, char *contents, Tree **tree, Tree **treestack,
+                               int &ntreestack, double *argstack, int &nargstack, int ivar, char *str, int &istr, char *&ptr)
 {
   double sx,sxx;
   double value,sy,sxy;
 
   // word is not a match to any special function
+  std::unordered_set<std::string> functions = {
```

Comment:
Could be `const`. Also, making it a `std::map<std::string,int>` would avoid a lot of ifs.